dashboard-api/docker-compose-common.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  cockroachdb:
+    image: cockroachdb/cockroach:v21.2.3
```

Comment:
Unless we are running a different version of `cockroach` on this project, at least on `fly` we are using `v20.1.4`, for example, https://github.com/vlab-research/fly/blob/b0e949944028edab871b895da10da49f02fb900e/devops/dev-cluster.sh#L31

Reply:
No need to use the same version. Actually, fly is behind due to an cockroach old bug that has since been fixed, but we need to do the upgrade.

Reply:
Okey, then I'll leave this as it is :)